spaces/S000051/README.md

```diff
@@ -0,0 +1,17 @@
+---
+uid: S000051
+name: Khalimsky line
+aliases:
+- Digital line
+refs:
+- doi: 10.1090/s0002-9939-99-04839-x
+  name: Cut-point spaces (Honari & Bahrampour)
+- doi: 10.32219/isms.80.1_15
+  name: On Generalized Digital Lines (Nakaoka, Tamari, Maki) 
+---
+
+The integers $\mathbb Z$ with basis $\{\{2i-1,2i,2i+1\}\mid i\in\mathbb Z\}\cup\{\{2i+1\}\mid i\in\mathbb Z\}$.
+
+It is the unique {P205} that is irreducibly so, that is, where no proper subset is itself a cut point space. See {{doi:10.1090/s0002-9939-99-04839-x}} for a proof.
```

Comment:
```suggestion
It is the unique {P205} such that no proper subspace is {P205}. See {{doi:10.1090/s0002-9939-99-04839-x}} for a proof.
```

I think "irreducibly" is distracting for the reasons noted, so I've suggested removing it.

Noting this fact seems in scope for $\pi$-Base; it's not impossible that someone might prove something is not a cut-point space by showing it's a subspace of S51.